### PR TITLE
Static resources localized, footer fixed.

### PR DESCRIPTION
**Pull request by abhn**
The footer is fixed. All static resources loaded from the cdns are now localized (for no good reason)

### CODE REVIEW

File: lumos/templates/home.html [hunk 1]
```diff
@@ -59,83 +57,83 @@ <h1 class="ui header">
...
+<footer id="footer">
 <div class="ui inverted vertical footer segment">
+    <div class="ui center aligned container">
     <p id='love'>Made with <i class="heart icon"></i>for DMCE</p>
     <div class="ui inverted section divider"></div>
     <div class="ui horizontal inverted small divided link list">
-        <a href="#" id="disclaimer" class="item" style="color: inherit;">Disclaimer</a>
-        <a class="item" id="feedback" class="item" style="color: inherit;">Feedback</a>
+        <a href="#" id="disclaimer" class="item" style="color: inhert;">Disclaimer</a>
+        <a class="item" id="feedback" class="item" style="color: inhert;">Feedback</a>
```
Comment (darth-d0d0) [inline lumos/templates/home.html]: http://www.w3schools.com/cssref/css_inherit.asp

inherit keyword

File: lumos/templates/home.html [hunk 2]
```diff
@@ -59,83 +57,83 @@ <h1 class="ui header">
...
       </div>
```
Comment (darth-dodo) [inline lumos/templates/home.html:85]: http://www.w3schools.com/cssref/css_inherit.asp
inherit keyword

File: lumos/templates/home2.html [hunk 3]
```diff
@@ -1,5 +1,4 @@
 <!DOCTYPE html>
-{% load staticfiles %}
 <html>
```
Comment (darth-dodo) [inline lumos/templates/home2.html]: Please read up on this

https://docs.djangoproject.com/en/1.9/howto/static-files/

File: lumos/templates/home2.html [hunk 4]
```diff
@@ -11,10 +10,10 @@
...
   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/1.11.8/semantic.min.css"/>
   <script src="https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/1.11.8/semantic.min.js"></script>
-    <script type="text/javascript" src="{% static 'js/feedback_form.js' %}"></script>
-    <link rel="stylesheet" href="{% static 'css/feedback_form.css' %}"/>
-    <script type="text/javascript" src="{% static 'js/home.js' %}"></script>
-    <link rel="stylesheet" href="{% static 'css/home.css' %}"/>
+    <script type="text/javascript" src="../static/js/feedback_form.js"></script>
+    <link rel="stylesheet" href="../static/css/feedback_form.css"/>
+    <script type="text/javascript" src="../static/js/home.js"></script>
+    <link rel="stylesheet" href="../static/css/home.css"/>
```
Comment (darth-dodo) [inline lumos/templates/home2.html:16]: https://docs.djangoproject.com/en/1.9/howto/static-files/

File: lumos/templates/home2.html (hunk 3 above)
Comment (darth-dodo) [inline lumos/templates/home2.html:2]: Please read up on this

https://docs.djangoproject.com/en/1.9/howto/static-files/

**Pull request merged**